samples/java-spring-tracing/src/it/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%date{ISO8601} %-5level %logger - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <logger name="akka" level="INFO"/>
+    <logger name="kalix" level="INFO"/>
+    <logger name="com.github.dockerjava" level="INFO"/>
+    <logger name="io.grpc.netty" level="INFO"/>
+    <logger name="org.testcontainers" level="INFO"/>
+
+    <root level="DEBUG">
```

Comment:
INFO?

Reply:
since this is for tests, I think DEBUG is fine.. right?